localization/i18n/de/OrcaSlicer_de.po

```diff
@@ -14260,7 +14263,7 @@ msgid "From Volumetric Speed"
 msgstr "Vom volumetrischen Speed"
 
 msgid "To Volumetric Speed"
-msgstr "zum volumetrischen Speed"
+msgstr "zur volumetrischer Geschwindigkeit"
```

Comment:
Unfortunately I'm not sure in wich context this expression is used but the current state of your change is grammatically not right.

This should say:
```suggestion
msgstr "zur volumetrischen Geschwindigkeit"
```